cst/plugins/eu.esdihumboldt.cst.functions.numeric/src/eu/esdihumboldt/cst/functions/numeric/sequentialid/SequentialID.java

```diff
@@ -88,9 +88,13 @@ protected Object evaluate(String transformationIdentifier, TransformationEngine
 			else {
 				try {
 					id = Integer.parseInt(startValue);
+					if (id == 0) {
```

Comment:
To get some background: Why is it not OK if the starting number is zero?

The condition will ignore values lower than zero - that seems to be not OK though according to the message below?

Reply:
After a few clarifications with Kate, Thorsten and Florian, I've got that ID should start with any integer larger than 0. No initial special characters are allowed. Also, IDs starting for example with 01, 001, 0001...should not be allowed because leading 0s are used only for padding.

Reply:
Exactly, leading 0es (padding) should not be added, since the primary use case was simple integer IDs. I think it can in principle also start with 0, but I think in GeoPackages I did indeed always start with 1.

Reply:
So maybe change the condition to `if (id < 0) {` ?

Reply:
Now there is the same condition as in the example generation. I check for any String, whose first character is 0, e.g. 0, 001, 0001 etc. They are all not allowed.